integrated_channels/blackboard/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+"""
+The Blackboard Integrated Channel package.
+"""
+
+__version__ = "0.0.1"
+
+default_app_config = ("integrated_channels.blackboard.apps."  # pylint: disable=invalid-name
```

Comment:
So, tell me about why this name ends in a dot and why we're disabling invalid-name linting?

Reply:
hah! it seems like we are splitting on multiple lines (why???) copy paste I think, removed it into one line good catch!